inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/dialect/MySQLDialect.java

```diff
@@ -156,9 +156,10 @@ public List<LogicalTypeRoot> unsupportedTypes() {
 
     @Override
     public PreparedStatement setQueryPrimaryKeySql(Connection conn, String tableIdentifier) throws SQLException {
-        PreparedStatement st = conn.prepareStatement(QUERY_PRIMARY_KEY_SQL);
-        st.setString(1, JdbcMultiBatchingComm.getDatabaseNameFromIdentifier(tableIdentifier));
-        st.setString(2, JdbcMultiBatchingComm.getTableNameFromIdentifier(tableIdentifier));
-        return st;
+        try (PreparedStatement st = conn.prepareStatement(QUERY_PRIMARY_KEY_SQL)) {
```

Comment:
@gong Hi, please take a look at this modification.

Reply:
I don't think here need to use try. AbstractJdbcDialect will close it.